tests/e2e/tidbcluster/tidbcluster.go

```diff
@@ -255,7 +255,7 @@ var _ = ginkgo.Describe("TiDBCluster", func() {
 	})
 
 	ginkgo.It("should upgrade TidbCluster with webhook enabled", func() {
-		tcCfg := newTidbClusterConfig(e2econfig.TestConfig, ns, "cluster", "admin", utilimage.TiDBV3Version)
+		tcCfg := newTidbClusterConfig(e2econfig.TestConfig, ns, "cluster", "admin", utilimage.TiDBV4)
```

Comment:
ditto.

Reply:
Please confirm the version changes again to make sure that the change does not break the upgrade case.